packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/legacy/styles.less

```diff
@@ -0,0 +1,9 @@
+.datacenter-backup-legacy {
+  .oui-icon-arrow-right_extra-thin {
```

Comment:
This can be moved to the top level so that it applies to all right arrow inside of the `backup` route